.github/workflows/python-wheels-publish-test.yml

```diff
@@ -62,6 +62,10 @@ jobs:
           CIBW_SKIP: "*-win32 *_i686"
           CIBW_TEST_SKIP: "*-macosx_universal2:arm64"
           CIBW_ENVIRONMENT: OPENEXR_RELEASE_CANDIDATE_TAG="${{ github.ref_name }}"
+          CIBW_BEFORE_BUILD: |
+            pip install tomli
```

Comment:
This is not needed. The build frontend (pip, build, etc) will automatically install the dependencies specified in the pyproject's build-system.requires section.